backend/data/scrapers/payload.py

```diff
@@ -7,10 +7,70 @@
 
 from data.config import LIVE_YEAR
 
-URL = "https://www.handbook.unsw.edu.au/api/es/search"
+import requests
+import json
+import time
+
+URL = "https://api-ap-southeast-2.prod.courseloop.com/publisher/browsepage-academic-items?"
 HEADERS = {
     "content-type": "application/json",
 }
+ITEMS_LIMIT = 20
+REQ_DELAY = 0.1
+
+def do_requests(content_type, items_per_req=ITEMS_LIMIT, max_items = 10000):
+    """
+    retuns a list of items.
+    """
+    offset = 0
+    items_list = []
+    while offset < max_items:
+        data_payload = _create_payload(offset, ITEMS_LIMIT, content_type)
+        r = requests.post(
+            URL,
+            data=json.dumps(data_payload),
+            headers=HEADERS,
+            timeout=60 * 5
+        )
+        # r.json() looks like { "data": { "data": [{}, ..., {}], "count": 123 } }
+        print(json.dumps(r.json())[:100], flush=True)
```

Comment:
debug print (mayb good to see that its actually working and not just silently failing)

Reply:
i wanna keep (ill add a string at the start tho) cos it helps to see the thing's working